__tests__/dotnet.test.ts

```diff
@@ -6,6 +6,20 @@ import { Platform } from '../src/platform';
 import { ToolsDirectory } from '../src/toolsDirectory';
 
 const targetDirectory = path.join('target', 'directory');
+const cakeVersion = '0.35.0';
+const installedCakeVersion = '0.34.0';
+const containsFileWithOldToolInstalled = (path: string) => {
+  switch(path)
```

Comment:
Missing space before parenthesis and invalid brace alignment.

```suggestion
  switch (path) {
```

Reply:
addressed.